apps/docs/pages/docs/api-reference/fields/external.mdx

```diff
@@ -611,4 +612,54 @@ const config = {
 }}
 />
 
+### `renderFooter(props)`
+
+Customize what will be displayed in the footer of the modal.
+
+```tsx {13} copy
+const config = {
+  components: {
+    Example: {
+      fields: {
+        data: {
+          type: "external",
+          fetchList: async () => {
+            return [
+              { title: "Hello, world", description: "Lorem ipsum 1" },
+              { title: "Goodbye, world", description: "Lorem ipsum 2" },
+            ];
+          },
+          renderFooter: ({ items }) => <span>{items.length} results</span>,
```

Comment:
Just realised it would probably help to demo something that looks different to the default. Not entirely sure what, but something like `<b>Custom footer with {items.length} results</b>`, or better